evaluation/plotting.py

```diff
@@ -12,20 +12,14 @@
 from matplotlib.transforms import Affine2D
 
 
-def save_figure(output_path: Path, format: str = "pdf"):
-    """Save the current figure to the provided output path."""
-    output_path.parent.mkdir(parents=True, exist_ok=True)
-    plt.savefig(output_path, bbox_inches="tight", format=format)
-
-
```

Comment:
The reason I wrapped the save_figure function is so that we can manage saved formats in one place. So can we bring it back and replace it with every `plt.savefig` function presented in this source file?